org.eclipse.scanning.event/src/org/eclipse/scanning/event/QueueReader.java

```diff
@@ -111,7 +111,7 @@ public QueueReader(IEventConnectorService service, Comparator<? super T> compara
 					try {
 						final T bean = (T)service.unmarshal(json, beanClass != null ? beanClass : statusBeanClass);
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unnecessary cast to "T". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1905)